package.json

```
@@ -10,7 +10,7 @@
   "api_cgn": "https://raw.githubusercontent.com/pagopa/io-backend/v7.29.0-RELEASE/api_cgn.yaml",
   "api_cgn_merchants": "https://raw.githubusercontent.com/pagopa/io-backend/v7.29.0-RELEASE/api_cgn_operator_search.yaml",
   "api_cgn_geo": "https://raw.githubusercontent.com/pagopa/io-backend/here_geoapi_integration/api_geo.yaml",
-  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.2/definitions.yml",
+  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/IOACGN-19-support-cgn-enable-feature-flag/definitions.yml",
```

```suggestion
  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.3/definitions.yml",
```
when the relative tag has been pushed